### PR TITLE
Added myself to the student section

### PR DESCRIPTION
**Pull request by abdulsamad234**
Added abdulsamad aliyu, github, image, twitter, facebook, blog, bio,
latitude and longitude to the students section.

Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

#Added image and credentials

- Added my info to the students section, live link: https://abdulsamad234.github.io/gci16.fossasia.org/, gci task link: https://codein.withgoogle.com/dashboard/task-instances/6566799275458560/.

### CODE REVIEW

File: _data/students.yml [hunk 1]
```diff
@@ -698,3 +698,13 @@
   bio: "Your average weeb and computer hobbyist. I use Emacs because it's moe."
   lat: -6.13698
   lng: 106.82279
+
+- name: Abdulsamad Aliyu
+  github: abdulsamad234
+  image: abdul.png
+  twitter: nitrocoder2
+  facebook: https://web.facebook.com/abdulsamad.aliyu
```
Comment (robbyoconnor) [inline _data/students.yml]: just your username here

**Review by robbyoconnor — CHANGES_REQUESTED** [review]

File: _data/students.yml (hunk 1 above)
Comment (abdulsamad234) [reply _data/students.yml]: in the facebook section?

**Review by robbyoconnor — APPROVED** [review]

**Comment by abdulsamad234** [comment]
I have made the changes

File: _data/students.yml (hunk 1 above)
Comment (robbyoconnor) [reply _data/students.yml]: fixed it

**Review by robbyoconnor — APPROVED** [review]

**Pull request merged**